Titan/Managers/ThreadManager.cs

```diff
@@ -302,7 +320,8 @@ public void StartMatchResolving(TitanAccount account, LiveGameInfo info)
                     var timeSpent = DateTime.Now.Subtract(account.StartEpoch.ToDateTime());
 
                     _log.Error("Connection to account {Account} timed out. It was not possible to resolve the Match " +
-                               "ID for the target after {Timespan} seconds.", account.JsonAccount.Username, timeSpent.Seconds);
+                               "ID for the target after {Timespan} seconds.", account.JsonAccount.Username,
+                        timeSpent.Seconds);
```

Comment:
Please fix formatting here.

Reply:
" // Will be edited by Cake Build Script "
Are you fucking gay or what?

Reply:
Yes, the value will be edited by the Cake Build Script but is supposted to be reverted back after the build process finished. It seems you cancelled the build process mid-way. 

Also, please follow the [Contributor Covenant Code of Conduct](https://github.com/Marc3842h/Titan/blob/master/CODE_OF_CONDUCT.md) before discussing and contributing to the Titan repository and the issues section. Inappropriate language may result in you losing your Collaborator permissions.